src/python/doc/rips_complex_tflow_itf_ref.rst

```diff
@@ -0,0 +1,44 @@
+:orphan:
+
+.. To get rid of WARNING: document isn't included in any toctree
+
+TensorFlow layer for Vietoris-Rips persistence
+##############################################
+
+.. include:: differentiation_sum.inc
+
+Example of gradient computed from Vietoris-Rips persistence
+-----------------------------------------------------------
+
+.. testsetup::
+    :hide:
```

Comment:
`:hide:` is invalid inside `testsetup`, and unnecessary because the code in `testsetup` is not shown anyway.